src/main/resources/swagger/api-docs.yaml

```diff
@@ -3251,6 +3251,33 @@ paths:
         - $ref: '#/parameters/workspaceNameParam'
         - $ref: '#/parameters/submissionIdParam'
         - $ref: '#/parameters/workflowIdParam'
+        - name: includeKey
+          description: >
+            When specified key(s) to include from the metadata. Matches any key starting with the value. May not be
```

Comment:
ah, these were copy-paste from Cromwell's swagger and I didn't read them very closely. Since this is user-facing I will take a shot at re-wording. Thanks!